tyr/servers/server.py

```diff
@@ -371,21 +386,58 @@ def blockdevicemapping(self):
 
         return bdm
 
+    def get_subnet_vpc_id(self, subnet_id):
+        vpcConn = VPCConnection()
+        subnets = vpcConn.get_all_subnets(
+            filters={'subnet_id': subnet_id})
+        if len(subnets) == 1:
+            vpc_id = subnets[0].vpc_id
+            return vpc_id
+        else:
+            self.log.error("None or greater then 1 subnet returned")
```

Comment:
For the `get_security_group_ids` method, there are multiple possible log statements - one for the case where there are no matching security groups and one for the case where there are multiple matching security groups. Perhaps it would be good to do that here also? That way when an error occurs, the user knows what the problem was, not that it was one of two possible problems.